qa/os/src/test/java/org/elasticsearch/packaging/util/ServerUtils.java

```diff
@@ -55,7 +55,7 @@
 
     private static final Logger logger = LogManager.getLogger(ServerUtils.class);
 
-    private static String SECURITY_ENABLED = "xpack.security.enabled: true";
+    private static String SECURITY_DISABLED = "xpack.security.enabled: false";
```

Comment:
Once this is merged, I'll open an issue to track the fact that we should change all the remaining packaging tests to work with security enabled ( which will be the default behavior ) . Given the fact that we will be making further changes to how elasticsearch starts in the context of the Security On by Default project and the changes this would mean for the packaging tests, it didn't make much sense to spend too much time to do this now.